pkg/universe.dagger.io/alpha/aws/cdk/python/image.cue

```diff
@@ -0,0 +1,57 @@
+package python
+
+import (
+	"universe.dagger.io/docker"
+)
+
+// Python image default name
+_#DefaultName: "python"
+
+// Python image default repository
+_#DefaultRepository: "index.docker.io"
+
+// Default python docker image version
+_#DefaultVersion: "3.8"
+
+#Image: {
+	name:       *_#DefaultName | string
+	repository: *_#DefaultRepository | string
```

Comment:
As I mentioned in other PR, those two fields are not useful :)

`#Image` stands for a default image, it user want to provide his own, we should take care of it to perfectly fit his need ;)